example/src/screens/NestedNavigator/ScreenWithLinks.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Button} from 'react-native';
+import {useNavigation} from '@react-navigation/native'
+
+export const ScreenWithLinks = (props: {links: { routeName: string}[]}) => {
+    const {navigate} = useNavigation()
+    return (<>
```

Comment:
just curious: what is the meaning of nameless tag `<>`? Is it used just to get into React world for parsing?

Reply:
This tag is the equivalent of `React.Fragment`. Components must have a single top level child, but when that does not make sense you can use one of them instead.